sys/amd64/start.S

```diff
@@ -0,0 +1,46 @@
+#include <amd64/asm.h>
+#define ASM_FILE
+#include <amd64/multiboot2.h>
+
+#define MULTIBOOT_HEADER_LENGTH (multiboot_header_end - multiboot_header_start)
+#define MULTIBOOT_HEADER_CHECKSUM \
+  (-(MULTIBOOT2_HEADER_MAGIC + MULTIBOOT_ARCHITECTURE_I386 + \
+     MULTIBOOT_HEADER_LENGTH))
+
+	.align MULTIBOOT_HEADER_ALIGN
+
+multiboot_header_start:
+
+	.long MULTIBOOT2_HEADER_MAGIC
+	.long MULTIBOOT_ARCHITECTURE_I386
+	.long MULTIBOOT_HEADER_LENGTH
+	.long MULTIBOOT_HEADER_CHECKSUM
+
+	# start of the tag list
+
+	# TODO(MichalBlk): request some information (e.g. meminfo, VBE...)
+
+	.align 8
+	.short MULTIBOOT_HEADER_TAG_FRAMEBUFFER
+	.short 0
+	.long 20    # sizeof(struct multiboot_header_tag_framebuffer)
+	.long 0     # no width preference
+	.long 0     # no height preference
+	.long 0     # no depth preference
+
+	# end of the tag list
+
+	.align 8
+	.short MULTIBOOT_HEADER_TAG_END
+	.short 0
+	.long 8     # sizeof(struct multiboot_tag)	
+
+multiboot_header_end:
+
+	.code32
+
+_ENTRY(_start)
+	jmp _start
+END(_start)
```

Comment:
I did this because the `asm.h` file that I've included doesn't supply the `_END()` macro. Relying on `aarch64/asm.h` I've added the macro by myself, so it's consistent with the rest of the asm files.